pkg/skaffold/config/options.go

```diff
@@ -51,22 +51,28 @@ type SkaffoldOptions struct {
 	RenderOnly            bool
 	ProfileAutoActivation bool
 	DryRun                bool
-	PortForward           PortForwardOptions
-	CustomTag             string
-	Namespace             string
-	CacheFile             string
-	Trigger               string
-	KubeContext           string
-	KubeConfig            string
-	WatchPollInterval     int
-	DefaultRepo           StringOrUndefined
-	CustomLabels          []string
-	TargetImages          []string
-	Profiles              []string
-	InsecureRegistries    []string
-	Command               string
-	RPCPort               int
-	RPCHTTPPort           int
+
+	// Avoids setting Skaffold-specific labels including runID, deployer labels, etc.
+	// `CustomLabels` are still applied. Must only be used in commands which don't deploy
+	// (e.g. `skaffold render`) since the runID label isn't available.
+	NoLabels bool
```

Comment:
As I mentioned elsewhere, try to keep bools in the positive. This should be "Labels" with a false, for instance.

`Labels` seems overly ambiguous here, given the huge amount of description required.

Reply:
Changed to `AddSkaffoldLabels bool`